visualizer/src/bin/etch_a_sketch.cpp

```diff
@@ -24,6 +25,9 @@ struct EtchaSketchSerialReciever : public serial::SerialReciever {
         EtchaSketchPotValNode* next = nullptr;
     };
 
+    /* Frequency at which to trace the etch a sketch */
+    float trace_freq = 1e4;
+
```

Comment:
this is the trace frequency that we get in from the keyboard. So for this case the full etch-a-sketch will be traced 1,000 times per second.